### ISSUE TITLE
add aria-labels to the terminal area

### ISSUE THREAD

**Issue by amisskii** [podman-desktop/podman-desktop#11953]
### Is your enhancement related to a problem? Please describe

The terminal area for containers/pods is missing ARIA labels, reducing accessibility and making it harder to locate during end-to-end tests

### Describe the solution you'd like

To enhance accessibility, aria-labels should be added there

**Comment by vancura** [comment]
Dupe of #12597
